cms/templates/widgets/_ui-dnd-indicator-after.html

```diff
@@ -1,2 +1,2 @@
 <%page expression_filter="h"/>
-<span class="draggable-drop-indicator draggable-drop-indicator-after"><i class="icon fa fa-caret-right"></i></span>
+<span class="draggable-drop-indicator draggable-drop-indicator-after"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
This file, and the 2 after, are used for drag and drop on the Studio course outline page. I'm thinking that sr text is needed. @clrux can you advise? I don't know if the drag and drop functionality is generally accessible.